application.rb

```diff
@@ -0,0 +1,25 @@
+require_relative 'dataset' # Data of countries
+require_relative 'country' # Country Class
+
+def main
+
+  countries_array = [] # array to store each country object
+
+  $countries_data.each do |country, data|
+    country_obj = Country.new(data[0],data[1],data[2],data[3],data[4],data[5])
```

Comment:
use splat operator or destructure the data

`    country_obj = Country.new(*data)`
or 

```
$countries_data.each do |country, (name, population, gdp, states, army_strength, state_of_country)|
  country_obj = Country.new(name, population, gdp, states, army_strength, state_of_country)
end
```